docs/usage.rst

```diff
@@ -411,6 +411,39 @@ Additional Notes
 
 Here are misc. notes about topics that are maybe not covered in enough detail in the usage section.
 
+Item flags
```

Comment:
well, create is a separate page and i find the main usage.rst to be a little too long now...